setup.py

```diff
@@ -17,7 +17,7 @@
 import os
 from setuptools import find_packages, setup
 
-with open(os.path.join(os.path.dirname(__file__), 'README.rst')) as readme:
+with open(os.path.join(os.path.dirname(__file__), 'docs/readme-pypi.rst')) as readme:
```

Comment:
I can't find this file. Where am I supposed to look?

I did:
```
$ git checkout develop
$ git pull
$ git grep pypi
```

Nothing relevant showed up.

Reply:
@OmarIthawi Dang! I forgot to add this file when I had to redo the branch. Added!